addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1401,8 +1401,11 @@ registry.WebsiteAnimate = publicWidget.Widget.extend({
             // not really one (eg 'discrete' layout), and should not be used as
             // scrollTop value.
             const $closestModal = $el.closest(".modal:visible");
-            const scrollTop = $closestModal[0] ?
-                $closestModal.scrollTop() : this.$scrollingElement.scrollTop();
+            const hasModalBackdrop = $closestModal[0] &&
+                !$closestModal[0].classList.contains("s_popup_no_backdrop");
+            const scrollTop = $closestModal[0] ? hasModalBackdrop ? $closestModal[0].scrollTop :
+                $closestModal[0].querySelector(".modal-content").scrollTop :
+                this.$scrollingElement[0].scrollTop;
```

Comment:
LGTM, Just a suggestion to make the diff more readable :+1:
```diff
-            const $closestModal = $el.closest(".modal:visible");
-            const scrollTop = $closestModal[0] ?
-                $closestModal.scrollTop() : this.$scrollingElement.scrollTop();      
+            const closestModal = $el.closest(".modal:visible")[0];
+            let scrollTop = this.$scrollingElement[0].scrollTop;
+            if (closestModal) {
+                scrollTop = closestModal.classList.contains("s_popup_no_backdrop") ?
+                    closestModal.querySelector(".modal-content").scrollTop :
+                    closestModal.scrollTop;
+            }
```

Reply:
Good call. :) It's done!